packages/theme-data/src/themes/mediumDensityTheme/system/density.js

```diff
@@ -1,62 +1,62 @@
 export default {
-  "SPACINGS.XXS": {
+  "spacings.xxs": {
     value: {
-      ref: "BASICS.SPACINGS.MEDIUM_XXS"
+      ref: "basics.spacings.mediumXxs"
     }
   },
-  "SPACINGS.XS": {
+  "spacings.xs": {
     value: {
-      ref: "BASICS.SPACINGS.MEDIUM_XS"
+      ref: "basics.spacings.mediumXs"
     }
   },
-  "SPACINGS.S": {
+  "spacings.s": {
     value: {
-      ref: "BASICS.SPACINGS.MEDIUM_S"
+      ref: "basics.spacings.mediumS"
     }
   },
-  "SPACINGS.M": {
+  "spacings.m": {
     value: {
-      ref: "BASICS.SPACINGS.MEDIUM_M"
+      ref: "BASICS.spacings.mEDIUM_M"
```

Comment:
`BASICS`  is uppercase in this file also.

Reply:
Updated in 4693f7d